rootfs/usr/local/bin/setup.sh

```diff
@@ -309,6 +309,20 @@ if [ -f /var/mail/postfix/custom.conf ]; then
   echo "[INFO] Custom Postfix configuration file loaded"
 fi
 
+#
+# If you defined whitelist for clients, you need to define
+# "rbl_override" file under "/etc/postfix" directory, then
+# you need to add the following settings:
+#     check_client_access hash:/etc/postfix/rbl_override
```

Comment:
I see that we adding this line unconditionally above. Will this work if there is no /etc/postfix/rbl_override file?

Reply:
I updated the script. Please take another look.